river/ensemble/aggregated_mondrian_forest.py

```diff
@@ -0,0 +1,444 @@
+from river.tree.mondrian_tree_classifier import MondrianTreeClassifier
+
+from abc import ABC, abstractmethod
+
+from river.base.classifier import Classifier
+
+
+class AMFLearner(ABC):
+    """
+    Base class for Aggregated Mondrian Forest classifier and regressors for online learning.
+
+    Note
+    ----
+    This class is not intended for end users but for development only.
+
+    """
+
+    def __init__(
+            self,
+            n_estimators: int,
+            step: float,
+            loss: str,
+            use_aggregation: bool,
+            split_pure: bool,
+            random_state: int,
+    ):
+        """Instantiates a `AMFLearner` instance.
+
+        Parameters
+        ----------
+        n_estimators : int
```

Comment:
No need to specify the type of each attribute. We determine the types from the type annotations in the `__init__` method.

Reply:
I think I fixed that one looking at the HoeffdingTree, please let me know if there's anything to change =)